StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1209UnitTests.cs

```diff
@@ -58,6 +60,31 @@ class A
     }
 }";
 
+            var fixedTestCodeCompilationUnit = @"using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+
+using TasksNamespace = System.Threading.Tasks;
+
+class A
+{
+}";
+
+            var fixedTestCodeNamespace = @"namespace Test
+{
+    using System.IO;
+    using System.Net;
+    using System.Threading;
+
+    using L = System.Linq;
+    using P = System.Threading.Tasks;
+
+    class A
+    {
+    }
+}";
+
             DiagnosticResult[] expectedForCompilationUnit =
             {
                 this.CSharpDiagnostic().WithLocation(1, 1)
```

Comment:
:bulb: Single diagnostics can be:
`var expected = this.CSharpDiagnostic().WithLocation(1, 1);`